.github/workflows/claude.yml

```
@@ -0,0 +1,28 @@
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  claude:
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name == 'pull_request') ||
+      (github.event_name == 'issue_comment' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)